src/ui/settings/index.js

```diff
@@ -29,7 +29,8 @@ export default function setupSettings(kbnServer, server, config) {
     return client
       .get({ ...clientSettings })
       .then(res => res._source)
-      .then(user => hydrateUserSettings(user));
+      .then(user => hydrateUserSettings(user))
+      .catch(e => { return {}; });
```

Comment:
Oh, how interesting! I tend to prefer the more explicit syntax so its obvious to the reader what's going on but this is good to know!